packages/mdc-drawer/README.md

```diff
@@ -60,21 +60,74 @@ npm install @material/drawer
 
 ### Drawers with separate list groups
 
-If you need multiple list groups within the drawer please see the [list readme](https://github.com/material-components/material-components-web/tree/master/packages/mdc-list#list-groups)
-for detailed use and instruction.
+```html
+<nav class="mdc-drawer">
+  <div class="mdc-drawer__content">
+    <nav class="mdc-list">
+      <a class="mdc-list-item mdc-list-item--activated" href="#" tabindex="0">
+        <i class="material-icons mdc-list-item__graphic" aria-hidden="true">inbox</i>
+        <span class="mdc-list-item__label mdc-drawer__list-item-label">Inbox</span>
+      </a>
+      <a class="mdc-list-item" href="#">
+        <i class="material-icons mdc-list-item__graphic" aria-hidden="true">star</i>
+        <span class="mdc-list-item__label mdc-drawer__list-item-label">Star</span>
+      </a>
+      <a class="mdc-list-item" href="#">
+        <i class="material-icons mdc-list-item__graphic" aria-hidden="true">send</i>
+        <span class="mdc-list-item__label mdc-drawer__list-item-label">Sent Mail</span>
+      </a>
+      <a class="mdc-list-item" href="#">
+        <i class="material-icons mdc-list-item__graphic" aria-hidden="true">drafts</i>
+        <span class="mdc-list-item__label mdc-drawer__list-item-label">Drafts</span>
+      </a>
+
+      <hr class="mdc-list-divider">
+      <h6 class="mdc-list-group__subheader mdc-drawer__list-group-subheader">Labels</h6>
+      <a class="mdc-list-item" href="#" tabindex="0">
+        <i class="material-icons mdc-list-item__graphic" aria-hidden="true">bookmark</i>
+        <span class="mdc-list-item__label mdc-drawer__list-item-label">Family</span>
+      </a>
+      <a class="mdc-list-item" href="#">
+        <i class="material-icons mdc-list-item__graphic" aria-hidden="true">bookmark</i>
+        <span class="mdc-list-item__label mdc-drawer__list-item-label">Friends</span>
+      </a>
+      <a class="mdc-list-item" href="#">
+        <i class="material-icons mdc-list-item__graphic" aria-hidden="true">bookmark</i>
+        <span class="mdc-list-item__label mdc-drawer__list-item-label">Work</span>
+      </a>
+    </nav>
+  </div>
+</div>
+```
+
+> NOTE: First navigation item in list group should be explicitely set tabindex to '0'.
+
+And then list group needs to be instantiated for keyboard navigation.
+
+```javascript
+var listList = document.querySelectorAll('.mdc-list');
+
+listList.forEach(function(list) {
```

Comment:
`(list)` -> `(el)` or `(listEl)` as with the demo above, to avoid redeclaring the same variable in the function